src/app/BUILD.gn

```diff
@@ -71,45 +77,33 @@ source_set("revision_info") {
   ]
 }
 
-source_set("app_config") {
-  sources = [ "AppConfig.h" ]
+source_set("path-params") {
+  sources = [
+    "AttributePathParams.h",
+    "EventPathParams.h",
+  ]
+}
 
-  deps = [ ":app_buildconfig" ]
+source_set("global-attributes") {
+  sources = [ "GlobalAttributes.h" ]
```

Comment:
depends at least on lib support (due to CodeUtils inclusion).

It also depends on zap-generated, however those are impossible to split out today.